tests/zfs-tests/tests/functional/alloc_class/alloc_class_013_pos.ksh

```diff
@@ -41,8 +41,7 @@ log_must display_status "$TESTPOOL"
 #
 
 log_must zfs create -o dedup=on -V 2G $TESTPOOL/$TESTVOL
-
-log_must eval "new_fs $ZVOL_DEVDIR/$TESTPOOL/$TESTVOL >/dev/null 2>&1"
+log_must eval "new_fs $ZVOL_DEVDIR/$TESTPOOL/$TESTVOL >/dev/null"
```

Comment:
This looks unrelated, but we should add a `block_device_wait $ZVOL_DEVDIR/$TESTPOOL/$TESTVOL` here to make sure the block device is available before using it.

Reply:
`block_device_wait $ZVOL_DEVDIR/$TESTPOOL/$TESTVOL` applied.

Not ignoring stderr gives us much better diagnostic. When the test case failed I had to wonder what were the reasons for file system creation failure and I ended up removing `>/dev/null`. I know it _might_ be obvious that it is due to missing block device, time saved on inserting diagnostics is time earned.